src/implemented.rs

```diff
@@ -16,11 +16,17 @@ impl Lerp for EaseValue<Sprite> {
                 (Some(a), None) => Some(a),
                 (Some(a), Some(b)) => Some(a.lerp(b, *scalar)),
             },
+            rect: match (self.0.rect, other.0.rect) {
+                (None, None) => None,
+                (None, Some(b)) => Some(b),
+                (Some(a), None) => Some(a),
+                (Some(a), Some(b)) => Some(EaseValue(a).lerp(&EaseValue(b), scalar).0),
+            },
             #[cfg(feature = "render")]
             color: EaseValue(self.0.color)
                 .lerp(&EaseValue(other.0.color), scalar)
                 .0,
-            ..Sprite::default()
+            ..other.0
```

Comment:
```suggestion
            ..Sprite::default()
```
It is not possible to use `other` as it's not `Copy`